atd-vzv/src/constants/time.js

```diff
@@ -1,6 +1,21 @@
 import moment from "moment";
 
-// Time data
+// Number of past years data to fetch
+// 4 full years of data, plus data up to the last complete month of the current year
+export const rollingYearsOfData = 4;
```

Comment:
food for thought... Since this is a value that shouldn't change, a true constant variable, we might consider using the all cap convention to signal it should not be altered in the program. 

https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/

Reply:
Yes, this is definitely the right choice here. I'll update before merge!